applications/nublado/values.yaml

```diff
@@ -41,8 +41,34 @@ controller:
   # set in the corresponding Nublado Vault secret.
   slackAlerts: false
 
+
   # Passed as YAML to the lab controller.
   config:
+  # The controller also manages file serving environments
+    fileserver:
+      # -- Enable fileserver management
+      # @default -- false
+      enabled: false
+
+      # -- Image for fileserver container
+      # @default -- None, must be specified
+      image: ghcr.io/lsst-sqre/worblehat
+
+      # -- Tag for fileserver container
+      # @default -- `latest`
+      tag: 0.1.0
+
+      # -- Pull policy for fileserver container
+      # @default -- `IfNotPresent`
+      pullPolicy: IfNotPresent
+
+      # -- Timeout for user fileservers, in seconds
+      # @default -- 3600
+      timeout: 64800
+
+      # -- Namespace for user fileservers
+      # @default -- `fileservers`
```

Comment:
`@default` is not something you should routinely set. You should only set it when the default isn't what's present in this file. It's the default from the perspective of someone writing a `values-<environment>.yaml` file. All of these `@default` lines should therefore be deleted (in some cases they're not correct since they don't match the actual default in this file).